packaging/msi/UI.wxs

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Wix
+    xmlns="http://wixtoolset.org/schemas/v4/wxs"
+    xmlns:ui="http://wixtoolset.org/schemas/v4/wxs/ui"
+    xmlns:util="http://wixtoolset.org/schemas/v4/wxs/util">
+    <Fragment>
+        <Property Id="UiFragment" Hidden="true"/>
+
+        <!-- Define the product icon -->
+        <Icon Id="icon.ico" SourceFile="!(bindpath.SOURCE_ROOT)packaging\msi\resources\product.ico"/>
+        <Property Id="ARPPRODUCTICON" Value="icon.ico" />
+
+        <!-- Set the default install folder -->
+        <Property Id="WIXUI_INSTALLDIR" Value="INSTALLFOLDER" />
+
+        <!-- Customize Exit Dialog -->
+        <Property Id="WIXUI_EXITDIALOGOPTIONALCHECKBOXTEXT" Value="View Release Notes" />
+        <Property Id="WIXUI_EXITDIALOGOPTIONALTEXT" Value="You need to reboot your computer to get the updated system path." />
+
+        <!-- Use the default WixUI_InstallDir dialog set and personalize the license an images -->
+        <WixVariable Id="WixUILicenseRtf" Value="!(bindpath.SOURCE_ROOT)packaging\msi\docs\LICENSE.rtf"/>
+        <WixVariable Id="WixUIBannerBmp" Value="!(bindpath.SOURCE_ROOT)packaging\msi\resources\top-banner.png"/>
+        <WixVariable Id="WixUIDialogBmp" Value="!(bindpath.SOURCE_ROOT)packaging\msi\resources\dialog.png"/>
+
+        <!-- This UI is and updated version of WixUI_InstallDir to use IceInstallDirDlg instead of the default InstallDirDlg
```

Comment:
Unfortunately customize the UI besides the built-in settings, requires quite a lot of duplication. For examples this wiring of dialogs is just to replace the default InstallDirDlg with the custom IceInstallDirDlg.